js/addPagination.js

```diff
@@ -0,0 +1,75 @@
+import {
+  LOAD_MORE_BUTTON,
+  SHOW_LESS_BUTTON,
+  CLICK_EVENT,
+  PAGINATED_NO,
+} from "/js/constants.js";
+
+let currentItem = PAGINATED_NO;
+const container = document.getElementById("container");
+const itemsToShow = PAGINATED_NO;
+let visibleItems = itemsToShow;
+
+LOAD_MORE_BUTTON.addEventListener(CLICK_EVENT, function (event) {
+  event.preventDefault();
+  let boxes = [...document.querySelectorAll(".container .content .item")];
```

Comment:
boxes isn't reused. So we can use `const`
```suggestion
  const boxes = [...document.querySelectorAll(".container .content .item")];
```

Reply:
sorry, I should've been more careful. done!